src/proposals/20201128/SparkEthereum_20241128.t.sol

```diff
@@ -0,0 +1,218 @@
+        DssAutoLineLike(AUTO_LINE).exec(ALLOCATOR_ILK);
+
+        base.selectFork();
+        // TODO: replace with deployed payload
+        payloadBase = deployPayloadBase();
+        mainnet.selectFork();
+    }
+
+    function testWBTCChanges() public {
+        ReserveConfig[] memory allConfigsBefore = createConfigurationSnapshot('', pool);
+        ReserveConfig memory wbtcConfig         = _findReserveConfigBySymbol(allConfigsBefore, 'WBTC');
+
+        assertEq(wbtcConfig.liquidationThreshold, 65_00);
+
+        executePayload(payload);
+
+        ReserveConfig[] memory allConfigsAfter = createConfigurationSnapshot('', pool);
+        wbtcConfig.liquidationThreshold        = 60_00;
+
+        _validateReserveConfig(wbtcConfig, allConfigsAfter);
+    }
+
+    function testcbBTCChanges() public {
+        ReserveConfig[] memory allConfigsBefore = createConfigurationSnapshot('', pool);
+        ReserveConfig memory cbBTCConfig        = _findReserveConfigBySymbol(allConfigsBefore, 'cbBTC');
+
+        assertEq(cbBTCConfig.liquidationThreshold, 70_00);
+        assertEq(cbBTCConfig.ltv, 65_00);
+
+        executePayload(payload);
+
+        ReserveConfig[] memory allConfigsAfter = createConfigurationSnapshot('', pool);
+        cbBTCConfig.liquidationThreshold       = 75_00;
+        cbBTCConfig.ltv                        = 74_00;
+
+        _validateReserveConfig(cbBTCConfig, allConfigsAfter);
+    }
+
+    function testExistingMorphoVault() public {
+        MarketParams memory sUSDeVault =  MarketParams({
+            loanToken:       Ethereum.DAI,
+            collateralToken: PT_SUSDE_27MAR2025,
+            oracle:          PT_27MAR2025_PRICE_FEED,
+            irm:             Ethereum.MORPHO_DEFAULT_IRM,
+            lltv:            0.915e18
+        });
+
+        _assertMorphoCap(sUSDeVault, 200_000_000e18);
+        executePayload(payload);
+        _assertMorphoCap(sUSDeVault, 200_000_000e18, 400_000_000e18);
+
+        skip(1 days);
+        IMetaMorpho(Ethereum.MORPHO_VAULT_DAI_1).acceptCap(sUSDeVault);
+        _assertMorphoCap(sUSDeVault, 400_000_000e18);
+    }
+
+    function testNewMorphoVault() public {
+        MarketParams memory USDeVault =  MarketParams({
+            loanToken:       Ethereum.DAI,
+            collateralToken: PT_USDE_27MAR2025,
+            oracle:          PT_27MAR2025_PRICE_FEED,
+            irm:             Ethereum.MORPHO_DEFAULT_IRM,
+            lltv:            0.915e18
+        });
+
+        _assertMorphoCap(USDeVault, 0);
+        executePayload(payload);
+        _assertMorphoCap(USDeVault, 0, 100_000_000e18);
+
+        skip(1 days);
+        IMetaMorpho(Ethereum.MORPHO_VAULT_DAI_1).acceptCap(USDeVault);
+        _assertMorphoCap(USDeVault, 100_000_000e18);
+    }
+
+    function testBridging() external {
+      uint256 baseBalanceBefore = 123496652107156694;
+      uint256 USDSMintAmount    = 90_000_000e18;
```

Comment:
can be declared as global var because it's used in several tests

Reply:
implemented in dba7bf8